gremlin-go/README.md

```diff
@@ -65,91 +58,222 @@ import (
 
 func main() {
 	// Creating the connection to the server.
-	driverRemoteConnection := gremlingo.NewDriverRemoteConnection("localhost", 8182)
+	driverRemoteConnection, err := gremlingo.NewDriverRemoteConnection("localhost", 8182)
+	if err != nil {
+		fmt.Println(err)
+		return
+	}
 	// Cleanup
-	defer driverRemoteConnection.Close()
-	// Submit a traversal (string format in milestone 1).
+	defer func(driverRemoteConnection *gremlingo.DriverRemoteConnection) {
+		err := driverRemoteConnection.Close()
+		if err != nil {
+			fmt.Println(err)
+			return
+		}
+	}(driverRemoteConnection)
+	
+	// Submit a simple string traversal
 	resultSet, err := driverRemoteConnection.Submit("1 + 1")
-	// Check for traversal execution errors, print them if there are any and exit.
```

Comment:
Was there a reason to remove this?

Reply:
Just the fact that it's obviously doing an error check...to declutter the lines.